### PR TITLE
Perform the initial setup for the localization

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
Addresses https://github.com/appium/appium-desktop/issues/818

de

### CODE REVIEW

File: app/configs/i18next.config.client.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import i18next from 'i18next';
```
Comment (dpgraham) [inline app/configs/i18next.config.client.js]: I'd rather this file be called `i18next.config.renderer.js`

File: locales/en/translation.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+  "en": "en",
```
[the remark this answers is not in the document]
Comment (mykola-mokhnach) [reply locales/en/translation.json]: yes, but later

**Comment by mykola-mokhnach** [comment]
main_win

**Comment by dpgraham** [comment]
Awesome!

File: app/configs/i18next.config.client.js (hunk 1 above)
Comment (mykola-mokhnach) [reply app/configs/i18next.config.client.js]: fixed

**Pull request merged**